src/EdgeConProblem/EdgeConReduction.c

```diff
@@ -178,138 +267,157 @@ static g_context_s* init_g_context(Z3_context z3_ctx, EdgeConGraph graph, int co
     return ctx;
 }
 
-
-static Z3_ast build_phi_2(const g_context_s *ctx){
-    int tab1Size = (ctx->m * (ctx->m - 1)) / 2;     // /!\ A REVOIR
-    Z3_ast resTab1[tab1Size];
+static Z3_ast build_phi_2_1(const g_context_s *ctx){
+    int nb_variables = (ctx->m * (ctx->m - 1)) / 2;
+    Z3_ast formulaTab[nb_variables];
```

Comment:
**question:** Why not initialize directly: 
```c
Z3_ast formulaTab[(ctx->m * (ctx->m - 1)) / 2];
```

**issue:** the size shouldn't be `ctx->N * (ctx->m * ctx->m - 1)`?